R/theme.r

```diff
@@ -192,12 +193,21 @@
 #' )
 #'
 #' # Axes ----------------------------------------------------------------------
+#' # Change styles of axes texts and lines
 #' p1 + theme(axis.line = element_line(size = 3, colour = "grey80"))
 #' p1 + theme(axis.text = element_text(colour = "blue"))
 #' p1 + theme(axis.ticks = element_line(size = 2))
-#' p1 + theme(axis.ticks.length = unit(.25, "cm"))
+#'
+#' # Change the appearance of the y-axis title
 #' p1 + theme(axis.title.y = element_text(size = rel(1.5), angle = 90))
 #'
+#' # Make ticks point outwards on y-axis and upwards on x-axis
```

Comment:
One small remaining nitpick: "inwards" instead of "upwards"? x-axis tick marks are always vertical, so arguably they always point up (and down).